README.md

```diff
@@ -161,3 +161,100 @@ deploy:
   on:
     tags: true
 ```
+
+## Running build_platforms.py locally
+1. Install arduino-cli from here: https://arduino.github.io/arduino-cli/installation/
+2. Download ci-arduino
+   * `git clone https://github.com/adafruit/ci-arduino`
+3. Put these lines at the end of your `.bashrc` or `.bash_profile` if you're on OSX. Make sure to fill in the path to where you installed ci-arduino and replacing USER with your username.
+   ```bash
+   alias test-platforms='python3 ~/path/to/ci-arduino/build_platform.py'
+   export HOME=/home/USER/
+   ```
+4. Run this at the top level of the library you want to test
+   ```bash
+   adafruit@adafruit:~/Adafruit_BMP183_Library$ export GITHUB_WORKSPACE=$(pwd)
+   ```
+5. Remove everything in test library, and re-create it
+   ```bash
+   adafruit@adafruit:~/Adafruit_BMP183_Library$ rm -rf ~/Arduino/libraries/Adafruit_Test_Library/; mkdir ~/Arduino/libraries/Adafruit_Test_Library
```

Comment:
There aren't any previous steps that mention test library, so what are we removing?

Reply:
If this isn't the first time it gets run, you need to remove it since there is likely already stuff in there which I was worried would confuse it.